lib/cmake/.conan-recipes/pybind11/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  2.10.0:
+    sha256: eacf582fa8f696227988d08cfc46121770823839fe9e301a20fbce67e7cd70ec
+    url: https://github.com/pybind/pybind11/archive/v2.10.0.tar.gz
```

Comment:
Where do the pybind11 files come from? How do we update in the future?

Reply:
I copied this from the official recipe. I hope by the time we need to update next the issue will be fixed upstream. We have not updated in years, so that might not be a bad bet.